docs/http-triggers.md

```diff
@@ -128,6 +129,37 @@ $ curl --data '{"Another": "Echo"}' \
 {"Another": "Echo"}
 ```
 
+To enable CORS on a HTTPTrigger there are following ways.
```

Comment:
Thanks for adding the docs! I would structure this a little bit different. I would create two different sections and I would add them just before the section `Enable Kong Security Plugins`:


## Enable CORS

It's possible to enable CORS requests at the HTTPTrigger level. To do so use the `--cors-enable` flag when deploying the HTTPTrigger or add the field `cors-enable: true` to the YAML manifest.

## Add arbitrary annotations

It is also possible to add any annotation to the resulting Ingress object if you add those to the HTTPTrigger. For example:

```console
apiVersion: kubeless.io/v1beta1
kind: HTTPTrigger
metadata:
 name: cors-trigger
 annotations:
  nginx.ingress.kubernetes.io/enable-cors: "true"
  nginx.ingress.kubernetes.io/cors-allow-methods: "GET"
spec:
 function-name: get-python
 host-name: example.com
 path: echo
```

The above will create an Ingress object with the annotations `nginx.ingress.kubernetes.io/enable-cors: "true"` and `nginx.ingress.kubernetes.io/cors-allow-methods: "GET"`.